src/AnimatedLineGraph.tsx

```diff
@@ -89,11 +89,11 @@ export function AnimatedLineGraph({
     })
 
     const previous = paths.current
-    let from: SkPath = previous.to ?? straightLine
+    let from: SkPath | null = previous.to ?? straightLine
     if (previous.from != null && interpolateProgress.current < 1)
       from = from.interpolate(previous.from, interpolateProgress.current)
 
-    if (path.isInterpolatable(from)) {
+    if (from && path.isInterpolatable(from)) {
```

Comment:
`from` can never be null because `straightLine` can never be null, so this can be removed